packages/amplication-data-service-generator/src/dsg-context.ts

```diff
@@ -1,11 +1,26 @@
 import * as types from "@amplication/code-gen-types";
+import { ContextUtil } from "@amplication/code-gen-types";
+import { join } from "path";
+import { readStaticModules } from "./read-static-modules";
+import winston from "winston";
 
-export class DsgContext implements types.DsgContext {
+const contextUtil = {
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+  setDsgPath: (path: string) => join(__dirname, `../node_modules/${path}`),
```

Comment:
We need to access "node_modules" because we are using `__dirname` in `dsg_context` via a wrapper function (setDsgPath).
If we will instruct the user to use __dirname directly in the plugin they will not need to access node_modules because the "static" folder is relative to their code

Reply:
remove the function